test/api/apicubeplot.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# Test the C API for reading via a matrix and making a plot
+ps=apicubeplot.ps
+if [ ! -f cube.nc ]; then
+	cp "${src}"/../grdinterpolate/cube.nc .
+fi
+testapi_cube 
```

Comment:
Like #4483, you have to add a wrapper for `testapi_cube` in `test/gmtest.in`.

Reply:
OK, did that now.